native_client/swift/deepspeech_ios/deepspeech.h

```diff
@@ -0,0 +1,357 @@
+#ifndef DEEPSPEECH_H
```

Comment:
woo, is that a copy of the root one? the naming of the directory would suggest no, but the filename and the ifdef would